system/jlib/jfile.cpp

```diff
@@ -4331,6 +4331,24 @@ IFileIOStream * createBufferedAsyncIOStream(IFileAsyncIO * io, unsigned bufsize)
     return new CBufferedAsyncIOStream(io, bufsize);
 }
 
+IFileIOStream * createIOStreamFromFile(const char *fileNameWithPath, IFOmode mode)
+{
+    Owned<IFile> wuInfoIFile = createIFile(fileNameWithPath);
+    Owned<IFileIO> wuInfoIO = wuInfoIFile->open(mode);
+    if (!wuInfoIO)
+        return nullptr;
+    return createIOStream(wuInfoIO);
+}
+
+IFileIOStream * createBufferedIOStreamFromFile(const char *fileNameWithPath, IFOmode mode, unsigned bufsize)
+{
+    Owned<IFile> wuInfoIFile = createIFile(fileNameWithPath);
+    Owned<IFileIO> wuInfoIO = wuInfoIFile->open(mode);
```

Comment:
minor: wuInfo* names should be changed, as this is a general function.

Reply:
Remove the "wu" from the names